pymc/tests/test_distributions_random.py

```diff
@@ -1321,6 +1321,18 @@ def check_basic_properties(self):
         assert np.all(draws <= 1)
 
 
+class TestStickBreakingWeights_1D_alpha(BaseTestDistributionRandom):
+    pymc_dist = pm.StickBreakingWeights
+    pymc_dist_params = {"alpha": [1.0, 2.0, 3.0], "K": 19}
+    expected_rv_op_params = {"alpha": [1.0, 2.0, 3.0], "K": 19}
+    sizes_to_check = [None]
```

Comment:
What about a couple more?
```suggestion
    sizes_to_check = [None, (3,), (5, 3)]
```

Reply:
Sure